types/react-relay/lib/relay-experimental/useBlockingPaginationFragment.d.ts

```diff
@@ -14,23 +14,30 @@ export interface ReturnType<TQuery extends OperationType, TKey, TFragmentData> {
 
 export type $Call<Fn extends (...args: any[]) => any> = Fn extends (arg: any) => infer RT ? RT : never;
 
-export type NonNullableReturnType<T extends { readonly ' $data'?: unknown }> = (arg: T) => NonNullable<T[' $data']>;
-export type NullableReturnType<T extends { readonly ' $data'?: unknown | null }> = (arg: T) => T[' $data'] | null;
+interface KeyType {
+    readonly ' $data'?: unknown;
+}
+
+type KeyReturnType<T extends KeyType> = (arg: T) => NonNullable<T[' $data']>;
```

Comment:
Why not export these?

Reply:
I just followed what was done in #40167, which also doesn't export these "helper" types.